src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -70,6 +75,27 @@ private JObject GetInvokeResult(byte[] script, IVerifiable checkWitnessHashes =
             return json;
         }
 
+        public static StackItem[] ConvertIEnumeratorToArray(StackItem[] stackItems)
```

Comment:
If we change the items in the same array, we should not return an array, because it's the same.